### PR TITLE
Implement renderers for chunk validation tracers

### PR DESCRIPTION
**Pull request by Jasagredo**
Propagates changes from https://github.com/input-output-hk/ouroboros-network/pull/3506 up to the node level.

### CODE REVIEW

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs [hunk 1]
```diff
@@ -490,7 +492,13 @@ instance ( ConvertRawHash blk
           "This block no longer in the VolatileDB and isn't in the ImmutableDB\
           \ either; it wasn't part of the current chain. Block: " <> renderRealPoint pt
         ChainDB.SwitchBackToVolatileDB ->  "SwitchBackToVolatileDB"
-      ChainDB.TraceImmutableDBEvent _ev ->  "TraceImmutableDBEvent"
+      ChainDB.TraceImmutableDBEvent ev -> case ev of
+        ImmDB.ChunkValidationEvent e -> case e of
+          ImmDB.ValidatingChunk c -> "Validating chunk no. " <> showT c
+          ImmDB.ValidatedChunk c c' ->
+            "Validated chunk no. " <> showT c <> " out of " <> showT c'
+            <> ". Progress: " <> pack (showFFloat (Just 2) (100 * fromIntegral (chunkNoToInt c) / fromIntegral (chunkNoToInt c') :: Float) "") <> "%"
```
Comment (dnadales) [inline cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs:500]: Could we shorten this line?

Comment (dnadales) [reply cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs:500]: Maybe we could use more descriptive names for `c` and `c'`.

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs [hunk 2]
```diff
@@ -16,6 +16,7 @@ module Cardano.Tracing.OrphanInstances.Consensus () where
 import           Cardano.Prelude hiding (show)
 import           Prelude (show)
 
+import           Numeric
```
Comment (dnadales) [inline cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs:19]: Should we import explicitly?

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs (hunk 1 above)
Comment (dnadales) [reply cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs:500]: What's the role of that `""`?

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs [hunk 3]
```diff
@@ -1296,10 +1296,10 @@ instance Show exception => ToObject (TraceLocalRootPeers RemoteAddress exception
              , "domainAddress" .= toJSON d
              , "diffTime" .= show dt
              ]
-  toObject _verb (TraceLocalRootResult d res) =
+  toObject _verb (TraceLocalRootResult d _) =
     mkObject [ "kind" .= String "LocalRootResult"
              , "domainAddress" .= toJSON d
-             , "result" .= Aeson.toJSONList res
+             , "result" .= String "Unk"
```
Comment (dnadales) [inline cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs:1302]: Why aren't we using `res` anymore?

**Review by dnadales — COMMENTED** [review]
LGTM, but I'll leave it up to the code owners to approve.

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs (hunk 3 above)
Comment (Jasagredo) [reply cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs]: This was from a leftover that was updated thanks to cherry-picking other commits from the Network team

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs (hunk 1 above)
Comment (Jasagredo) [reply cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs]: Reworded this into a function. The `""` at the end is becasue `showFFloat` is `StringS`

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs [hunk 4]
```diff
@@ -441,6 +449,14 @@ instance ( ConvertRawHash blk
         ChainDB.ScheduledGC slot _difft ->
           "Scheduled a garbage collection for " <> condenseT slot
       ChainDB.TraceOpenEvent ev -> case ev of
+        ChainDB.StartedOpeningDB ->
+          "Started opening db"
+        ChainDB.StartedOpeningImmutableDB ->
+          "Started opening immutable db"
```
Comment (newhoggy) [inline cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs:455]: I noticed that [[3388](https://github.com/input-output-hk/cardano-node/pull/3388](https://github.com/input-output-hk/cardano-node/pull/3388/files#r760611056) says "Started opening Immutable DB" whilst this PR says "Started opening immutable db".

Perhaps the casing should be the same as the other?

Similarly for other strings.

**Comment by Jasagredo** [comment]
Closing as this is subsumed by #3388

**Pull request closed without merging**